server/src/resolvers/args/ProgramingLanguageArgs.ts

```diff
@@ -0,0 +1,34 @@
+import { ArgsType, Field } from 'type-graphql';
+
+@ArgsType()
+export class getLanguageByIdArgs {
```

Comment:
Modify all "language" to "programmingLanguage" for args too

Reply:
Done